.circleci/config.yml

```diff
@@ -243,19 +276,14 @@ jobs:
     executor: default-machine
     steps:
       - checkout
-      - run:
-          name: Setup Docker env variables
-          command: |
-            echo 'export DOCKER_ORG=`cat package.json | jq -r .name | sed s/@// | sed s/\\\\/.*//`' >> $BASH_ENV
-            echo 'export CIRCLE_PROJECT_REPONAME=`cat package.json | jq -r .name | sed s/@// | sed s/\.*\\\\///`' >> $BASH_ENV
       - run:
           name: Build Docker local image
           command: |
             echo "Building Docker image: local"
-            docker build -t $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:local .
+            docker build -t ${DOCKER_ORG:-mojaloop}/$CIRCLE_PROJECT_REPONAME:local .
```

Comment:
whats the reason for this?

Reply:
The DOCKER_ORG variable is organisation level variable in circleCI that doesn't propagate incase of forked PR's. So I just want to pass a default value if that variable is not set.